client/src/Tools/FetchData/fetchContestsfunctions.js

```diff
@@ -0,0 +1,16 @@
+import { requestOptionsGET } from './requestOptions';
+
+export async function getContestsCards(taker) {
+  let url = '/api/contests/card/data';
+  if (taker) {
```

Comment:
Can you explain me the functionality of the taker?

Reply:
I couldn't find better word for "biorca / adresat / odbiorca" - the "person" who receives that data (maybe receiver?). Depends on which role is chosen at frontend Contests Page shows contests which are sellected differently. I made one endpoint for Contests Page and added this query to differentiate, which variant should be given.
When contest page is rendered for Manager Role --> it should show just contests where logged user is added as manager. For Staff Role --> to not make a mess, are shown just present contests. Etc.

Reply:
I think you could also use filtering on the page for that, so they could decide for themselves which constest they want to see exactly. Thanks for the explanation!

Reply:
![image](https://user-images.githubusercontent.com/58532638/159126507-64237e2a-f6e7-4b37-907f-67cbf00310d4.png)
So this is what I get when I click "zaplanowane zawody". I think we could put a title for it somwhere or info (you are now viewing contests planned by you or stg like that)

Reply:
added to TODOs